environment.yml

```diff
@@ -37,4 +37,4 @@ dependencies:
 - typing-extensions~=4.0
 - upf_to_json~=0.9.2
 - wrapt~=1.11
-- chardet~=5.2.0  # [win]
```

Comment:
This was correct, but the script that autogenerates this file does not support environment markers. I'll open an issue about that.

Reply:
I think we'll need to expand the script that autogenerates this file to handle these markers. Hopefully shouldn't be hard to do, I can take a crack at it.

Reply:
But it shouldn't be a blocker for this PR, it just means that there's gonna be one extra package installed for windows afaik.